_episodes/01-motivation.md

```diff
@@ -7,118 +7,108 @@ questions:
-- Branches help keeping the project organized.
+- Often you need to work on several issues in one code - without branching this can be messy and confusing.
+- You can simulate branching by copying the entire code to multiple places but also this will be messy and confusing.
 
 
 ### Collaboration
 
-- No more *"can you please send me the latest version?"*.
-- Allows up to thousands of persons to work on the same project without stepping on each other's toes.
-- Distributed version control enables to get contributions from people you
-  don't know yet and contribute to projects maintained by people who don't know
-  you.
+- *"I will just finish my work and then you can start with your changes."*.
+- *"Can you please send me the latest version?"*.
+- *"Where is the latest version?"*.
+- *"Which version are you using?"*.
+- *"Which version have the authors used in the paper I am trying to reproduce?"*.
 
 
 ### Reproducibility
 
-- When you publish a paper you can indicate which version exactly was used so
-  others can reproduce the results.
-- When you find a bug, you can find out when precisely this bug was introduced
-  (are published results affected? has the bug made it to the last release of your code?).
+- How do you indicate which version of your code you have used in your paper?
+- When you find a bug, how do you know when precisely this bug was introduced
+  (are published results affected? do you need to inform collaborators or users of your code?).
+
+
+### What is the problem with this kind of "version control"?
+
+Discuss the following directory listing:
+
+```shell
+mylib-1.2.4_18.3.07.tgz         somecode_CP_10.8.07.tgz
+mylib-1.2.4_27.7.07.tgz         somecode_CP_17.5.07.tgz
+mylib-1.2.4_29.4.08.tgz         somecode_CP_23.8.07_final.tgz
+mylib-1.2.4_6.10.07.tgz         somecode_CP_24.5.07.tgz
+mylib-1.2.5_23.4.08.tgz         somecode_CP_25.5.07.tgz
+mylib-1.2.5_25.5.07.tgz         somecode_CP_29.5.07.tgz
+mylib-1.2.5_6.6.07.tgz          somecode_CP_30.5.07.tgz
+mylib-1.2.5_bexc.tgz            somecode_CP_6.10.07.tgz
+mylib-1.2.5_d0.tgz              somecode_CP_6.6.07.tgz
+mylib-1.3.0_4.4.08.tgz          somecode_CP_8.6.07.tgz
+mylib-1.3.1_4.4.08.tgz          somecode_KT.tgz
+mylib-1.3.2_22.4.08.tgz         somecode_PI1_2007.tgz
+mylib-1.3.2_4.4.08.tgz          somecode_PI_2007.tgz
+mylib-1.3.2_5.4.08.tgz          somecode_PI2_2007.tgz
+mylib-1.3.3_1.5.08.tgz          somecode_PI_CP_18.3.07.tgz
+mylib-1.3.3_20.5.08.tgz         somecode_11.5.08.tgz
+mylib-1.3.3_tstrm_27.6.08.tgz   somecode_15.4.08.tgz
+mylib-1.3.3_wk_10.8.08.tgz      somecode_17.6.09_unfinished.tgz
+mylib-1.3.3_wk_11.8.08.tgz      somecode_19.7.09.tgz
+mylib-1.3.3_wk_13.8.08.tgz      somecode-20.7.09.tgz
+...
+```
 
 ---
 
-## Why Git?
+## Compare with Dropbox or Google Drive
+
+- Document/code is in one place, no need to email snapshots.
+- Possible to get old versions but in a much less useful way - snapshots of files, not directories.
+
```

Comment:
- How can you use an old version? (add to second point)
- What about if you want to work on multiple versions at the same time?  do you make a copy?  how do you merge copies?
- You rely on dropbox/google - what if you don't have internet?